examples/light-switch-app/silabs/efr32/src/AppTask.cpp

```diff
@@ -226,6 +226,11 @@ void AppTask::SwitchActionEventHandler(AppEvent * aEvent)
 {
     if (aEvent->Type == AppEvent::kEventType_Button)
     {
+
+#if CHIP_DEVICE_CONFIG_ENABLE_SED == 1
+        DeviceLayer::ConnectivityMgr().RequestSEDCheckinMsg();
```

Comment:
DeviceLayer is the wrong layer for this. DeviceLayer cannot do Matter messages directly, only deal with events that may cause it to be done. Suggest figuring-out a better layer for this

Reply:
It's not the intention of the DeviceLayer to sent the Checkin message, but to trigger whatever entity that will actually send the checkin message. Said entity isn't yet determined nor implemented. Also since most of the SED specific APIs are driven by the ConnectivityManager, it seemed righteous to pursuit in the same direction instead of dispersing SED API all over the stack.

Reply:
DeviceLayer _will not be able to get its hands_ on the right entity in the end.  So we will need to funnel this through something else.